src/travel-details-screens/DepartureDetailsScreenComponent.tsx

```diff
@@ -211,47 +230,46 @@ export const DepartureDetailsScreenComponent = ({
                 interactiveColor={ctaColor}
               />
             )}
-            {shouldShowMapButton || realtimeText ? (
-              <View style={styles.headerSubSection}>
-                {realtimeText && !activeItem.isTripCancelled && (
-                  <LastPassedStop realtimeText={realtimeText} />
-                )}
-                {shouldShowMapButton ? (
-                  <Button
-                    type="small"
-                    leftIcon={{svg: Map}}
-                    style={realtimeText ? styles.liveButton : undefined}
-                    text={t(
-                      vehiclePosition
-                        ? DepartureDetailsTexts.live(t(translatedModeName))
-                        : DepartureDetailsTexts.map,
-                    )}
-                    interactiveColor={interactiveColor}
-                    onPress={() => {
-                      vehiclePosition &&
-                        analytics.logEvent(
-                          'Departure details',
-                          'See live bus clicked',
-                          {
-                            fromPlace: mapData.start,
-                            toPlace: mapData?.stop,
-                            mode: mode,
-                            subMode: subMode,
-                          },
-                        );
-                      onPressDetailsMap({
-                        legs: mapData.mapLegs,
-                        fromPlace: mapData.start,
-                        toPlace: mapData.stop,
-                        vehicleWithPosition: vehiclePosition,
-                        mode: mode,
-                        subMode: subMode,
-                      });
-                    }}
-                  />
-                ) : null}
+            {shouldShowMapButton ? (
+              <View style={styles.actionButtons}>
+                <Button
+                  type="small"
```

Comment:
We don't have the favorite button yet, but it looks like the map button should fill the width of its container, which I guess the `"small"` button doesn't currently support 🤔 Might make more sense to have width as it's own parameter to `Button` instead of being decided by `type`.

Maybe a problem for another issue/PR, but we should figure out if this solution is ok to release if it isn't solved before then.

Reply:
Yes, I had it before with a new attribute called "expand" but it seems like it can be messy for the design system and such, this is going to be in a different PR.